tests/integration/test_sanity.py

```diff
@@ -642,3 +642,19 @@ def test_metadata_restore_after_restart(cluster):
     # n2 triggered an election, so n2 should have voted for itself.
     assert n2.info()['raft_current_term'] == 2
     assert n2.info()['raft_voted_for'] == 2
+
+
+def test_followers_in_oom(cluster):
+    cluster.create(3)
+
+    cluster.node(2).execute('config', 'set', 'maxmemory', 1)
+    cluster.node(3).execute('config', 'set', 'maxmemory', 1)
+
+    cluster.execute("set", "abc", 1234)
+    cluster.wait_for_unanimity()
+
+    assert cluster.execute("get", "abc") == b'1234'
+    node2 = cluster.node(2)
+    node3 = cluster.node(3)
+    assert node2.execute("raft.debug", "exec", "get", "abc") == b'1234'
+    assert node3.execute("raft.debug", "exec", "get", "abc") == b'1234'
```

Comment:
```suggestion
    assert cluster.node(2).raft_debug_exec("get", "abc") == b'1234'
    assert cluster.node(3).raft_debug_exec("get", "abc") == b'1234'
```

Just an alternative to make linter happy. If needed, I guess even we can rename `raft_debug_exec()` -> `debug_exec()`